salt/config/__init__.py

```diff
@@ -1241,6 +1243,7 @@ def _gather_buffer_space():
     'id_function': {},
     'cachedir': os.path.join(salt.syspaths.CACHE_DIR, 'minion'),
     'append_minionid_config_dirs': [],
+    'worker_threads': 3,
```

Comment:
Are we certain about this? This default has been `5` for a long time. Could you explain this change more, please?

Reply:
It was so for master I'm not sure what's the best one for minion. I would change it to 5.